src/core/DataFile.cpp

```diff
@@ -1100,14 +1107,15 @@ void DataFile::upgrade_1_3_0()
 						attribute.setAttribute( "value", "MultibandLimiter" );
 					}
 
-					// Handle port changes
-
+					/*
+					 * CALF port changes
+					 */
 					if( attribute.attribute( "name" ) == "plugin" &&
 							( attribute.attribute( "value" ) == "MultibandLimiter" ||
 							attribute.attribute( "value" ) == "MultibandCompressor" ||
 							attribute.attribute( "value" ) == "MultibandGate" ) )
 					{
-						auto fn = [&](QDomElement& port, int num, QList<QDomElement>&, QList<QDomElement>& removeList)
+						auto fn = [&](QDomElement& port, int, int num, QList<QDomElement>&, QList<QDomElement>& removeList)
```

Comment:
Isn't there a name for the unidentified int value. Nothing major but for readability sake.

Reply:
In my opinion, not useful. This would only be relevant to the caller of that lambda, which is the common routine `iterate_ladspa_ports`. However, at this place, we do not need to know what to pass. So I would prefer to resolve these comments without change, unless you see a strong reason.

Reply:
Ohh ok then.